### DISCUSSION TITLE
post/make-http-client-design-flexible/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#52] [Announcements]
# post/make-http-client-design-flexible/

통신 실패 및 다양한 시나리오에 대응하여 직관적이고 일관성 있게 제어할 수 있는 설계 방법을 다룹니다.

https://tech.kakaopay.com/post/make-http-client-design-flexible/

**Comment by holyachon** [comment]
좋은 글 잘 읽었습니다!

[MSA 환경에서의 오류 전달 및 핸들링](
https://tech.kakaopay.com/post/make-http-client-design-flexible/?giscus=10ff8ca8e4afd526a74b01f5q6owXFT9Qzd2NCVVFWfwD9zquI3VeKUxtfjLSGSW2DGRQ9cCqdVUZa8Zk71VxzwPiUAHv03oNpUrkp33kSoDxPxaYb4wwffLHoL0XZkwBfoZ1TkpAuimZyQTIFI%3D#msa-%ED%99%98%EA%B2%BD%EC%97%90%EC%84%9C%EC%9D%98-%EC%98%A4%EB%A5%98-%EC%A0%84%EB%8B%AC-%EB%B0%8F-%ED%95%B8%EB%93%A4%EB%A7%81) 에 나오는 B API Sample Code 주석에 ‘객체’ 가 ‘객채’ 로 오타가 있어 알려드립니다 :)

**Comment by jikimee64** [comment]
글 잘 읽었습니다!!  자바 기반인 MSA는 어떻게 설계해야 할까요? ㅠ

**Comment by K-J-HYEON** [comment]
글 잘읽었습니다! 빨리 서비스 회사로 이직해서 포스팅과 같은 코드 설계경험도 해보고 싶네요.
